Cargo.toml

```diff
@@ -32,12 +32,24 @@ optional = true
 version = "1.0"
 default-features = false
 
+[dependencies.rkyv]
+optional = true
+version = "0.7"
+default-features = false
+
+[dependencies.bytecheck]
+optional = true
+version = "0.6"
+
 [dependencies.rand]
 optional = true
 version = "0.8"
 default-features = false
 
 [features]
 default = ["std"]
-std = ["num-traits/std"]
+std = ["num-traits/std", "dep:bytecheck"]
```

Comment:
This "dep:" format won't work with older MSRV, but I wonder why you have `std` enabling this in the first place?

It seems like `bytecheck` should remain an optional dependency of its own, since AFAICT `rkyv` doesn't even need it unless you turn on validation. Although that gets messy if a third party enables that validation feature, I suppose.

Reply:
I removed bytecheck from std features list :+1: